components/benefit_sponsors/app/controllers/benefit_sponsors/application_controller.rb

```diff
@@ -110,19 +116,27 @@ def page_alphabets(source, field)
       #   |object| object["_id"]
       # end
       # but source.collection acts on the entire collection (Model.all) hence cant be used here as source is a Mongoid::Criteria
-    source.distinct(field).collect {|word| word.first.upcase}.uniq.sort
-    rescue
+      source.distinct(field).collect {|word| word.first.upcase}.uniq.sort
+    rescue StandardError
       ("A".."Z").to_a
     end
 
+    def grouped_alphabet(source, field)
+      letter_groups = [("A".."F").to_a,("G".."L").to_a,("M".."R").to_a, ("S".."Z").to_a]
+      existing_letters = source.distinct(field).collect {|word| word.first.upcase}.uniq.sort
+      letter_groups.select {|group| (group & existing_letters).any?}
+    rescue StandardError
+      [("A".."F").to_a,("G".."L").to_a,("M".."R").to_a, ("S".."Z").to_a]
+    end
+
     def create_sso_account(user, personish, timeout, account_role = "individual")
-      if !user.idp_verified?
+      unless user.idp_verified?
         IdpAccountManager.create_account(user.email, user.oim_id, stashed_user_password, personish, account_role, timeout)
         session[:person_id] = personish.id
         session.delete("stashed_password")
         user.switch_to_idp!
       end
-      #TODO TREY KEVIN JIM CSR HAS NO SSO_ACCOUNT
+      #TODO: TREY KEVIN JIM CSR HAS NO SSO_ACCOUNT
```

Comment:
todo

Reply:
this just popped up from my prettifier, not a pertinent change